### PR TITLE
fixed wrong return type in xxhash

### LINKED ISSUE #5200: wrong return type in xxhash intdigest and hexdigest ?
```python
from xxhash import xxh32_hexdigest, xxh32_intdigest

result = xxh32_hexdigest(b"input") # (variable) result: bytes
print(type(result)) # <class 'str'>

result = xxh32_intdigest(b"input") # (variable) result: bytes
print(type(result)) # <class 'int'>
```

### PR DESCRIPTION
**Pull request by woodsman1**
Fixes #5200 wrong return type in xxhash `intdigest` and `hexdigest`. Now the return types are `int` for `intdigest` and `str` and `bytes` for `hexdigest` on Python 3 and Python 2 respectively.

### CODE REVIEW

File: stubs/xxhash/xxhash.pyi [hunk 1]
```diff
@@ -20,17 +20,25 @@ class xxh3_64(_IntDigestHash): ...
...
+    def xxh32_hexdigest(input: ReadableBuffer = ...) -> str: ...
+    def xxh64_hexdigest(input: ReadableBuffer = ...) -> str: ...
+    def xxh3_64_hexdigest(input: ReadableBuffer = ...) -> str: ...
+    def xxh3_128_hexdigest(input: ReadableBuffer = ...) -> str: ...
+
+else:
+    def xxh32_hexdigest(input: ReadableBuffer = ...) -> bytes: ...
+    def xxh64_hexdigest(input: ReadableBuffer = ...) -> bytes: ...
+    def xxh3_64_hexdigest(input: ReadableBuffer = ...) -> bytes: ...
+    def xxh3_128_hexdigest(input: ReadableBuffer = ...) -> bytes: ...
```
Comment (Akuli) [inline stubs/xxhash/xxhash.pyi:41]: On Python 2, `bytes` and `str` are the same:

```suggestion
def xxh32_hexdigest(input: ReadableBuffer = ...) -> str: ...
def xxh64_hexdigest(input: ReadableBuffer = ...) -> str: ...
def xxh3_64_hexdigest(input: ReadableBuffer = ...) -> str: ...
def xxh3_128_hexdigest(input: ReadableBuffer = ...) -> str: ...
```

**Review by Akuli — CHANGES_REQUESTED** [review]

**Comment by Akuli** [comment]
You seem to be new to making pull requests on github, so I'll explain how to  proceed:
- I suggested changes to this pull request. If you click "Commit suggestion", then my changes are a part of this PR on github, but git on your computer doesn't know it. So you will need to `git pull` after committing the suggestion on github.
- You can add more changes to your pull request by committing and pushing. They will appear here.

**Comment by Akuli** [comment]
Nit: Now that the `hexdigest` functions are just like all others, I don't think it makes sense to move them all to the end.

**Review by Akuli — APPROVED** [review]

**Comment by woodsman1** [comment]
Thank you for helping me through this entire process.

**Pull request merged**